app/src/features/apiClient/screens/apiClient/components/clientView/components/request/components/KeyValueTable/KeyValueTableWIthAutoGeneratedValues.tsx

```diff
@@ -0,0 +1,44 @@
+import { KeyValueFormType, KeyValuePair, RQAPI } from "features/apiClient/types";
+import { KeyValueTable } from "./KeyValueTable";
+import { isEmpty, noop } from "lodash";
+import { EnvironmentVariables } from "backend/environment/types";
+import React from "react";
+
+interface KeyValueTableProps {
+  data: KeyValuePair[];
+  autoGeneratedValues: KeyValuePair[];
+  pairType: KeyValueFormType;
+  variables: EnvironmentVariables;
+  setKeyValuePairs: (updaterFn: (prev: RQAPI.Entry) => RQAPI.Entry) => void;
+}
+
+const KeyValueTableWIthAutoGeneratedValues: React.FC<KeyValueTableProps> = ({
+  data,
+  autoGeneratedValues,
+  setKeyValuePairs,
+  variables,
+  pairType,
+}) => {
+  return (
+    <div className="keyvalue-table-wrapper">
+      {!isEmpty(autoGeneratedValues) && (
+        <KeyValueTable
+          data={autoGeneratedValues}
```

Comment:
Hiding the table autogenerated headers is missing right now

Reply:
DIdn't have the proper design for it. Will release in next version